### PR TITLE
Add documentation to Profile.ts

### PR DESCRIPTION
**Pull request by leihelen**
I added comments to some of the functions in Profile.ts to help with documentation.

### CODE REVIEW

File: client/public/index.html [hunk 1]
```diff
@@ -50,7 +50,7 @@
 </head>
 
 <body>
-  <div id="render-target" class="full-height"></div>
+  <div id="render-target" class="full-height">Please enable javascript to view this page.</div>
```
Comment (michelleli01) [inline client/public/index.html:53]: Did you mean to make this change?

File: README.md [hunk 2]
```diff
@@ -116,6 +116,8 @@ You will probably need to run this for the `classes`, `subjects` and `reviews` c
 - **Michelle Li** - Developer Advisor
 - **Harrison Chin** - Developer
 - **Samuel Yeboah** - Developer
+- **Helen Lei** - Developer
```
Comment (michelleli01) [inline README.md:119]: A minor thing: maybe specify that this PR is also adding people to the README or make it a separate PR

File: server/endpoints/Profile.ts [hunk 3]
```diff
@@ -9,10 +9,14 @@ export interface NetIdQuery {
   netId: string;
 }
 
+// The type of a query with a token
 export interface ProfileRequest {
   token: string;
 }
 
+/**
+ * [getStudentEmailByToken] returns a student email if the given token is valid
```
Comment (michelleli01) [inline server/endpoints/Profile.ts:18]: This is my bad for not specifying the requirements of a doc string. But, ideally we would have something like this:
```
/**
   * Returns true if [insert something here] matching the email if the given token is valid and false otherwise.
   * This method authenticates the user token through the Google API.
   * @param token: google auth token
   * @requires that you have a handleVerifyError, like as follows:
   * verify(token, function(){//do whatever}).catch(function(error){
   */
```

Something similar to the code I wrote above to specify parameters, return types, etc. Granted this file doesn't do that well or consistently hence the new documentation requirement

File: server/endpoints/Profile.ts [hunk 4]
```diff
@@ -36,7 +40,7 @@ export const getStudentEmailByToken: Endpoint<ProfileRequest> = {
 };
 
 /**
- * Counts the number of reviews made by a given student id.
+ * [countReviewsByStudentId] returns the number of reviews made by a given student id.
```
Comment (michelleli01) [inline server/endpoints/Profile.ts:43]: See above.

**Review by michelleli01 — CHANGES_REQUESTED** [review]
Great work! Sorry about the unclear instructions for the dev portfolio assignment. I left some comments :)

File: server/endpoints/Profile.ts [hunk 5]
```diff
@@ -9,10 +9,14 @@ export interface NetIdQuery {
...
+// The type of a query with a token
 export interface ProfileRequest {
   token: string;
 }
 
+/**
+ * [getStudentEmailByToken] returns a student email if the given token is valid
+ */
 export const getStudentEmailByToken: Endpoint<ProfileRequest> = {
   guard: [body("token").notEmpty().isAscii()],
```
Comment (michelleli01) [inline server/endpoints/Profile.ts:21]: I hint I wanted to add especially since the method CU Reviews uses to create APIs is unique and kinda weird, if you want to check the parameters taken in, check the `guard` variable and take a look at want is required. So for example here in this `getStudentEmailByToken` function, a `token` variable is required and is being checked to see if it's empty and an ASCII value. This is ensure types are rigid and there isn't much room for error.

**Comment by wizhaaa** [comment]
I think you may have accidentally created this branch off of your other branch in your other PR (adding to README branch)

Next time, make sure to switch back to the master branch, run git pull to get any updates off of the remote repository, and then create a new branch from the master branch.
